src/components/repl/Repl.tsx

```diff
@@ -38,8 +32,8 @@ import OutputConsole from './OutputConsole';
 import ReplToolbar from './ReplToolbar';
 import ShowFigures from './ShowFigures';
 
-const OutlineWrapper = styled(Box)(
-  sx({
+const OutlineWrapper = styled(Box)(({ theme }) =>
+  theme.unstable_sx({
```

Comment:
What is `unstable_sx` as opposed to `sx`?

Reply:
It lets you use the [`sx` syntax inside the styled utility or directly inside the theme](https://mui.com/material-ui/customization/theme-components/#the-sx-syntax-experimental). It just got renamed from `experimental_sx` imported from `@mui/material` [see before the change](https://github.com/graasp/graasp-app-code-capsule/blob/bee611c7a08cbc6763fd247fd459926f36bfeff5/src/components/repl/Repl.tsx#L12) to `theme.unstable_sx`